ops-bedrock/docker-compose.yml

```diff
@@ -89,7 +89,6 @@ services:
       - "7302:7300"
     environment:
       OP_PROPOSER_L1_ETH_RPC: http://l1:8545
-      OP_PROPOSER_L2_ETH_RPC: http://l2:8545
```

Comment:
This is directly causing the devnet to fail. You did not remove this flag as a required flag despite it no longer being used. Note that fully removing the flag requires more work across different repositories & should be called out in the PR description.